packages/usa-form/src/styles/_usa-form.scss

```diff
@@ -17,41 +17,13 @@
     text-decoration: none;
   }
 
-  .usa-input,
-  .usa-range,
-  .usa-select,
-  .usa-textarea {
-    // max width not needed since .usa-form has one of its own
+  // Clear max-width in children since .usa-form has one of its own.
+  // Used :where() to reduce specificity and
```

Comment:
Does this create an implicit requirement that the `usa-form` styles have to be loaded before the `usa-input` styles, since they're still equal specificity? (e.g. if a project has an [optimized build](https://designsystem.digital.gov/documentation/migration/#6-optimize-your-installation:~:text=Load%20the%20your%20project%E2%80%99s%20necessary%20component%20packages%20in%20your%20Sass%20entry%20point.%20If%20you%20find%20a%20hit%20for%20the%20class%20name%20in%20your%20codebase%2C%20include%20the%20relevant%20package%20in%20your%20Sass%20entry%20point.))

Reply:
Good catch @aduth 

After some quick testing in sandbox, it does look like loading `usa-form` after `usa-input` _will override_ the utility class with the `max-width: none` set here.  

Since the specificity is lower it also allows for other utility classes to override the max width easier.

Experimented with moving the following into various orders:

```sss
@forward "usa-form";
@forward "usa-input";
@forward "usa-select";
@forward "uswds-utilities";
```

|  Forward order | Develop | Working branch |
|--------|--------|--------|
| form, input, select, utilities | ![image](https://github.com/user-attachments/assets/7195c352-de5c-4105-a0d8-163371e2be5c) | ![image](https://github.com/user-attachments/assets/75beee42-02bc-4c3b-83b9-d52989442ffb) | 
| input, form, select, utilties | ![image](https://github.com/user-attachments/assets/092c7c87-45b7-412b-83a9-d79541748f2a) | ![image](https://github.com/user-attachments/assets/941c0707-b1b0-4ea4-8d49-1ef0118c8b5f) | 
| select, form, input, utilities | ![image](https://github.com/user-attachments/assets/1d5748ba-c885-4894-953d-25698042b8cb) | ![image](https://github.com/user-attachments/assets/7937b260-d686-4cf1-9ce8-c3c7da9c02e5) | 
| utilities, form, input, select | ![image](https://github.com/user-attachments/assets/dfd12dfc-9278-46f2-86e5-8e865d271c38) | ![image](https://github.com/user-attachments/assets/336bd6ba-e462-4dd0-895f-882109db73b9) | 
| utilities, input, select, form | ![image](https://github.com/user-attachments/assets/c75c9e48-c539-4ef6-b56b-c649477c22e7) | ![image](https://github.com/user-attachments/assets/7b7309f2-9927-49b5-b8fd-a3e347e91f1e) | 

Findings:

- In `develop`
  - The high specificity within `usa-form` made things consistent no matter what order the packages were forwarded
  - Max width utility could not penetrate styles
- With this change:
  - Forward order does matter
  - Form being imported first seems to be the best option for versatility
  - Forwarding form after input _or_ select packages means the form max-width will take priority
  - Forwarding form after `uswds-utilities` classes means the form max-width will take priority
  - Forwarding form last means the form max-width will take priority over all of the above

I'm going to look at this again on Monday along side things like `uswds-form-controls` which imports all of the following:

```
usa-character-count, usa-checkbox, usa-combo-box, usa-date-picker, usa-error-message, usa-fieldset, usa-file-input, usa-form, usa-form-group, usa-hint, usa-input, usa-input-prefix-suffix, usa-label, usa-legend, usa-memorable-date, usa-radio, usa-range, usa-select, usa-textarea, usa-time-picker
```

Reply:
Good notes. 

To improve resiliency and address this issue, I've updated the Sass to generate both `.usa-form .usa-input--[width]` and `.usa-input--[width]` in [d9e1740](https://github.com/uswds/uswds/pull/6232/commits/d9e1740129c63bdf7b70b9d37af939b40fab606d) (same for usa-input-group). I also removed the `where()` from `usa-form` in [d6d2eb4](https://github.com/uswds/uswds/pull/6232/commits/d6d2eb40cf437ffa01ea527f9f6057de3bda688a) because it was no longer needed. 

Let me know if you have questions!

Reply:
Continuing my testing, it looks like any time a component that extends `%block-input-` placeholders from `_form.scss` is forwaded _before_ `usa-form`, it causes the `max-width: none` definition to take priority.

This appears to be true even for something like `usa-checkbox`:

**Importing usa-form first:**

```scss
@forward "usa-form";
@forward "usa-checkbox";
@forward "usa-input";
```

Results in:

![image](https://github.com/user-attachments/assets/f0ab788e-802b-4c44-bc52-62c769e6f3ee)

**Importing some other stylesheet with `%block-input-` placeholder first:**

```scss
@forward "usa-checkbox";
@forward "usa-form";
@forward "usa-input";
```

![image](https://github.com/user-attachments/assets/644aaa24-9e92-4c00-89da-438b49a6e9d7)

I'm a bit puzzled by this tbh but can spend more time diving in if we consider this a blocker for release.

**note:** As you can see in these screenshots, the form max-width classes have gone back to taking priority over our utility classes. This is the current behavior on `develop` (see table screenshots in [this comment](https://github.com/uswds/uswds/pull/6232#discussion_r1874142139))

Reply:
Update: I restored `where()` in `usa-form` to reduce specificity on those max-width rules. It looks like it solves the issue. Let me know if it gives you any trouble, @mahoneycm.

Reply:
This is looking great! From my testing, the package forward/import order no longer affects which `max-width` takes priority. This branch now matches behaviors found in `develop`. 

Great work @amyleadem !